test/e2e/extension-state.spec.ts

```diff
@@ -41,6 +43,14 @@ describe('Check correct app state persists after closing the extension', () => {
     await driver.quit()
   })
 
+  beforeAll(async () => {
+    server.listen(test.test.mockPort)
+  })
+
+  afterAll(async () => {
+    await closeServerAndWait()
+  })
```

Comment:
chore (blocking): spoke about with @dalebennett1992 on slack, but moving to a single test setup file